src/main/steps/tasklistresponse/respond-to-allegations-of-harm/respondToAOHMapper.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/no-unused-vars*/
+import { CaseWithId } from '../../../app/case/case';
+import { PartyDetails } from '../../../app/case/definition';
+
+export const prepareRespondToAOHRequest = (userCase: CaseWithId): Partial<PartyDetails['response']> => {
+  return {
+    responseToAllegationsOfHarmYesOrNoResponse: userCase.aoh_wishToRespond,
+    respondentResponseToAllegationOfHarm: userCase.aoh_responseToAllegations,
+  };
+};
+
+export const mapResponseToAOH = (partyDetails: PartyDetails): Partial<CaseWithId> => {
+  return {
+    aoh_wishToRespond: partyDetails.response.responseToAllegationsOfHarmYesOrNoResponse,
+    aoh_responseToAllegations: partyDetails?.response?.respondentResponseToAllegationOfHarm ?? '',
+  };
+};
```

Comment:
Do you need this specific file? given they are tiny methods, can they be addressed in the same util file? or is it just the convention we are following?

Reply:
It's more of a convention we follow, so that dev know that any request preparation / response transformation is done in the respective module mapper file.

Reply:
It's more of a convention we follow, so that dev know that any request preparation / response transformation is done in the respective module wrapper.